### PR TITLE
SNOW-2702: Remove duplicate table creation in snow-share sql script

### PR DESCRIPTION
**Pull request by govindunravel**
[SNOW-2702](https://unraveldata.atlassian.net/browse/SNOW-2702)

Remove duplicate table creation in snow-share sql script

[SNOW-2702]: https://unraveldata.atlassian.net/browse/SNOW-2702?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: script/snow_share/snow_share_procedure.sql [hunk 1]
```diff
@@ -826,13 +798,7 @@ GRANT SELECT ON TABLE METERING_DAILY_HISTORY to share S_SECURE_SHARE;
 GRANT SELECT ON TABLE METERING_HISTORY to share S_SECURE_SHARE;
 GRANT SELECT ON TABLE DATABASE_REPLICATION_USAGE_HISTORY to share S_SECURE_SHARE;
 GRANT SELECT ON TABLE REPLICATION_GROUP_USAGE_HISTORY to share S_SECURE_SHARE;
-GRANT SELECT ON TABLE DATABASE_STORAGE_USAGE_HISTORY to share S_SECURE_SHARE;
-GRANT SELECT ON TABLE STAGE_STORAGE_USAGE_HISTORY to share S_SECURE_SHARE;
-GRANT SELECT ON TABLE SEARCH_OPTIMIZATION_HISTORY to share S_SECURE_SHARE;
-GRANT SELECT ON TABLE DATA_TRANSFER_HISTORY to share S_SECURE_SHARE;
-GRANT SELECT ON TABLE AUTOMATIC_CLUSTERING_HISTORY to share S_SECURE_SHARE;
```
Comment (viswa-unravel) [inline script/snow_share/snow_share_procedure.sql:833]: Why deleting this ?

Comment (viswa-unravel) [reply script/snow_share/snow_share_procedure.sql:833]: all 5 is unique right ?

File: script/snow_share/snow_share_procedure.sql [hunk 2]
```diff
@@ -194,15 +172,9 @@ replicateData("METERING_DAILY_HISTORY", true, "USAGE_DATE");
 replicateData("METERING_HISTORY", true, "START_TIME");
 replicateData("DATABASE_REPLICATION_USAGE_HISTORY", true, "START_TIME");
 replicateData("REPLICATION_GROUP_USAGE_HISTORY", true, "START_TIME");
-replicateData("DATABASE_STORAGE_USAGE_HISTORY", true, "USAGE_DATE");
```
Comment (viswa-unravel) [inline script/snow_share/snow_share_procedure.sql:197]: Why deleting here

**Review by viswa-unravel — CHANGES_REQUESTED** [review]
Seems you are deleting many correct tables also

File: script/snow_share/snow_share_procedure.sql (hunk 2 above)
Comment (govindunravel) [reply script/snow_share/snow_share_procedure.sql:197]: below also same, pls check line 178.

File: script/snow_share/snow_share_procedure.sql (hunk 1 above)
Comment (govindunravel) [reply script/snow_share/snow_share_procedure.sql:833]: duplicate only deleted

**Review by viswa-unravel — APPROVED** [review]
LGTM

**Pull request merged**